src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2001,16 +2001,7 @@ private void InternalSaveAs(string path, SaveContext saveContext, bool isBackup
             try
             {
                 Model.Logger.Log(String.Format(Properties.Resources.SavingInProgress, path));
-
-                // If the current workspace is a CustomNodeWorkspaceModel, then call the save method on it.
-                if (CurrentSpaceViewModel.Model is CustomNodeWorkspaceModel)
-                {
-                    CurrentSpaceViewModel.Model.Save(path, false, Model.EngineController);
-                }
-                else
-                {
-                    CurrentSpaceViewModel.Save(path, isBackup, Model.EngineController, saveContext);
-                }
+                CurrentSpaceViewModel.Save(path, isBackup, Model.EngineController, saveContext);
```

Comment:
So for none custom node workspace, including saveContext does not matter and would not change the save file result?

Reply:
yeah no change in result. This save method was called with saveContext before this issue came up.